src/sinks/prometheus/exporter.rs

```diff
@@ -151,16 +152,44 @@ struct PrometheusExporter {
     server_shutdown_trigger: Option<Trigger>,
     config: PrometheusExporterConfig,
     metrics: Arc<RwLock<IndexSet<MetricEntry>>>,
-    last_flush_timestamp: Arc<RwLock<i64>>,
     acker: Acker,
 }
 
+#[derive(Debug)]
+struct MetricEntry {
```

Comment:
Consider naming it `ExpiringMetric` or similar to avoid collision with the `MetricEntry` above.

Reply:
Thanks for the good suggestion @bruceg! Will rename and will use `MetricEntry` from buffer as inner.